clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/DcosServerGroupNameResolver.groovy

```diff
@@ -40,11 +40,11 @@ class DcosServerGroupNameResolver extends AbstractServerGroupNameResolver {
     }
 
     def filteredApps = apps.findAll {
-      DcosSpinnakerAppId.parse(it.id).get().serverGroupName.cluster == Names.parseName(clusterName).cluster
+      DcosSpinnakerAppId.parse(it.id, true).get().serverGroupName.cluster == Names.parseName(clusterName).cluster
```

Comment:
Yeah, that would've been nicer, but I don't really wanna go back and refactor that quite yet XD.
We can do that as apart of the inevitable refactoring of the load balancer ID when the new load balancer comes around for 1.10, I can add a TODO with those functions just so we have some documentation to remind us to go do that.